lib/webdriveragent.js

```diff
@@ -262,6 +266,19 @@ class WebDriverAgent {
     return agentUrl;
   }
 
+  async startiproxy () {
+    return await new B(async (resolve, reject) => {
+      this.iproxy.on('exit', (code) => {
+        if (code && code !== 0) {
```

Comment:
If `code` is 0 then the first check will fail. That is to say, `if (code)` is functionally the same as `if (code && code !== 0)`